src/lib.rs

```diff
@@ -207,15 +207,7 @@ impl fmt::Display for Error {
         write!(f, "Error during a `trash` operation: {self:?}")
     }
 }
-impl error::Error for Error {
-    fn source(&self) -> Option<&(dyn error::Error + 'static)> {
-        match self {
-            #[cfg(all(unix, not(target_os = "macos"), not(target_os = "ios"), not(target_os = "android")))]
-            Self::FileSystem { path: _, source: e } => e.source(),
```

Comment:
Isn't this the point of the PR?

Reply:
It seems like this implementation should be kept, it was added in the commit before this one. All other style changes, especially those of imports (`fs::foo` instead of `std::fs::foo`, etc), should be removed for consistency with the rest of the codebase though.